practices/java-web/HelloWorldFilterWebApplication/src/main/java/com/dowa/java/web/StatisticsFilter.java

```diff
@@ -0,0 +1,38 @@
+package com.dowa.java.web;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+public class StatisticsFilter implements Filter {
+
+    private int contadorVisitasIndex = 0;
```

Comment:
too specific, how would you build this in a way that is generic, what we want is that if I add a new servlet the filter still works without changes